src/schema/text_options.rs

```diff
@@ -92,7 +165,7 @@ impl TextOptions {
 }
 
 #[derive(Clone, PartialEq, Debug, Eq, Serialize, Deserialize)]
-struct TokenizerName(Cow<'static, str>);
+pub struct TokenizerName(Cow<'static, str>);
```

Comment:
Should it be `pub(crate)` ? 
I don't think we use it public API do we?

Reply:
it's not reexported. I changed it to `pub(crate)`